src/CodeGenHelpers/EnumValueBuilder.cs

```diff
@@ -14,12 +14,27 @@ internal EnumValueBuilder(string name, EnumBuilder builder, int? value)
             Enum = builder;
         }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
+
         public string Name { get; }
 
         public int? Value { get; private set; }
 
         public EnumBuilder Enum { get; }
 
+        public EnumValueBuilder WithSummary(string summary)
+        {
+            XmlDoc.Summary = summary;
+            return this;
+        }
+
+        public EnumValueBuilder WithInheritDoc(bool inherit = true, string from = null)
+        {
+            XmlDoc.InheritDoc = inherit;
+            XmlDoc.InheritFrom = from;
+            return this;
+        }
```

Comment:
I can't say I see a use case where you would use InheritDoc on an Enum Value

```cs
public enum SomeEnum
{
    // Can't be inherited. 
    SomeValue
}
```

Reply:
You can inherit from another enum value. For example:
```csharp
public enum MyEnum
{
	/// <summary>
	/// Some documentation
	/// </summary>
	Value1,

	/// <inheritdoc cref="Value1"/>
	Value2,
}
```